src/containers/Tenant/Query/QueryEditor/QueryEditor.tsx

```diff
@@ -375,22 +365,12 @@ function QueryEditor(props: QueryEditorProps) {
                 </div>
                 <div className={b('pane-wrapper')}>
                     <Result
-                        executeQueryData={executeQueryResult.data}
-                        executeQueryError={executeQueryResult.error}
-                        cancelQueryError={cancelQueryResult.error}
-                        cancelQueryData={cancelQueryResult.data}
-                        explainQueryData={explainQueryResult.data}
-                        explainQueryError={explainQueryResult.error}
-                        explainQueryLoading={explainQueryResult.isLoading}
-                        executeResultLoading={executeQueryResult.isLoading}
-                        cancelQueryLoading={cancelQueryResult.isLoading}
                         resultVisibilityState={resultVisibilityState}
                         onExpandResultHandler={onExpandResultHandler}
                         onCollapseResultHandler={onCollapseResultHandler}
-                        onStopButtonClick={handleStopButtonClick}
                         type={type}
                         theme={theme}
-                        resultType={resultType}
+                        result={executeQuery.result}
```

Comment:
Let's add a key to `Result`
```suggestion
                        key={executeQuery.result?.queryId}
                        result={executeQuery.result}
```